featurebyte/routes/deployment/controller.py

```diff
@@ -201,6 +201,28 @@ async def compute_online_features(
         assert result is not None, result
         return result
 
+
+class AllDeploymentController(
```

Comment:
Created separate deployment controller to handle non catalog-specifc requests (e.g. list all deployments and deployments summary)

Reply:
🤔 i wonder if there's anything we can/should do to prevent catalog specific controllers/services from accessing all*controllers/services.